### ISSUE TITLE
Improve Benchmarks?

### ISSUE THREAD

**Issue by pyrossh** [kemalcr/kemal#308]
Your benchmarks are only against ruby which in itself is an interpreted language and will be slow. So you should be actually comparing it to another compiled language like go with fasthttp framework or rust with rocket webframework? And also you should take concurrency into picture.

**Comment by sdogruyol** [comment]
I agree with you that the benchmarks could be improved. There's this https://github.com/sdogruyol/kemal-showdown/pull/5

**Comment by YukiSakuraVN** [comment]
We already had Tech Empower benchmark. I read the code and see they implemented very well :)

**Comment by sdogruyol** [comment]
TechEmpower benchmarks: https://github.com/TechEmpower/FrameworkBenchmarks/tree/master/frameworks/Crystal/kemal
